### PR TITLE
Merge Avatar component

### PR DESCRIPTION
**Pull request by Hamdrive**
Created 5 types of avatars:
1. Round avatar with icon
2. Round avatar with user initials
3. Round avatar with user image
4. Square avatar with user image
5. Round corners avatar with user image

Sizes created: sm, md, lg, xl

Black borders only added for illustration of shape of avatar, not actually pushed into CSS

Avatars

### CODE REVIEW

File: stylesheets/reset.css [hunk 1]
```diff
@@ -53,6 +53,14 @@ h1 {
   font-size: 2rem;
 }
 
+img, picture, video, canvas,iframe, svg {
+  object-fit: cover;
+  width: 100%;
+  height: 100%;
+  display: block;
+
```
Comment (ritik-patel05) [inline stylesheets/reset.css]: remove empty space

**Review by ritik-patel05 — COMMENTED** [review]
The code looks clean and can be merged!

File: stylesheets/reset.css (hunk 1 above)
Comment (Hamdrive) [reply stylesheets/reset.css]: Prettier smh, will resolve

**Review by prachi-sahani — COMMENTED** [review]
code looks clean. Good to merge

**Review by NtshVrm — APPROVED** [review]

**Comment by Hamdrive** [comment]
Reviewed and changes done, merging branch to dev

**Pull request merged**